doc/user_manual/PostProcessors/SparseSensingdocumentation.tex

```diff
@@ -0,0 +1,60 @@
+\subsubsection{SparseSensing}
+\label{SparseSensing}
+The \textbf{SparseSensing} post-processor incorporates "PySensors", a Scikit-learn style Python package for the sparse placement of sensors for reconstruction tasks and classification tasks which will be added here soon.
```

Comment:
"PySensors" should probably be ``PySensors''  The other uses of quotation marks in the references should also be changed.

Reply:
Change the quotation marks.